app/services/metrics_service.rb

```diff
@@ -77,5 +77,17 @@ def time(metric, &block)
     def timing(namespace, difference)
       GovukStatsd.timing(namespace, difference)
     end
+
+    def gauge(stat, metric)
+      statsd.gauge(stat, metric)
+    end
+
+    def statsd
```

Comment:
It would be good to name this in some way to identify why it should be used compared to the `GovukStatsd` client.

Reply:
Thanks @cbaines , I have updated this in https://github.com/alphagov/email-alert-api/pull/1073/commits/6c284698227eb0b3f380f036e4839051beee0382